messaging/src/main/java/org/frankframework/extensions/mqtt/MqttFacade.java

```diff
@@ -71,9 +71,6 @@ public void configure() throws ConfigurationException {
 		if (StringUtils.isEmpty(getBrokerUrl())) {
 			throw new ConfigurationException("brokerUrl must be specified");
 		}
-		if (StringUtils.isEmpty(getTopic())) {
-			throw new ConfigurationException("topic must be specified");
-		}
```

Comment:
Deze check moet nu wel in de `MqttListener` gedaan worden.

Reply:
setTopic heeft de @Mandatory annotation waardoor dat automatisch door het framework afgevangen wordt.

Reply:
Nvm, ik dacht dat dit geimplementeerd was, maar dat is helaas niet zo! Wel een goeie feature suggestie